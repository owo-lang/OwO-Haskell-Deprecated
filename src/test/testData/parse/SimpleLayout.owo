module SimpleLayout where

topDef = 233
