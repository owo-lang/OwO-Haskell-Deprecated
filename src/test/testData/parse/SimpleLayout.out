File type: owo
Top module name: .SimpleLayout
 implementation of function topDef (27 3 1) (33 3 7)
  no pragmas
  clause for topDef (27 3 1) (33 3 7)
  patterns matching parsed as expression
   named reference topDef (27 3 1) (33 3 7)
  function body
   constant (36 3 10) (39 3 13)
    IntegerLit 233
  no with abstraction
  no where clause
